geode-core/src/main/java/org/apache/geode/internal/cache/TombstoneService.java

```diff
@@ -1078,8 +1087,8 @@ private void checkOldestUnexpired(long now) {
           if (logger.isTraceEnabled(LogMarker.TOMBSTONE_VERBOSE)) {
             logger.trace(LogMarker.TOMBSTONE_VERBOSE, "oldest unexpired tombstone is {}", oldest);
           }
-          long msTillHeadTombstoneExpires = oldest.getVersionTimeStamp() + EXPIRY_TIME - now;
-          if (hasExpired(msTillHeadTombstoneExpires)) {
+          long msUntilHeadTombstoneExpires = oldest.getVersionTimeStamp() + EXPIRY_TIME - now;
```

Comment:
Minor pick: if you rename the variable to `msUntilHeadTombstoneExpires`, it is better to rename the variable at other places, e.g. at line 759 as well, in order to be consistent. It is better for code maintenance.

Reply:
Done